systemtests/init_test.go

```diff
@@ -34,6 +34,21 @@ type systemtestSuite struct {
 	// user       string
 	// password   string
 	// nodes      []string
+	acinfo ACInfo
+}
+type ACInfo struct {
+	IP                string `json:"IP"`
+	HostIPs           string `json:"HOSTIPS"`
+	HostUsernames     string `json:"HOSTUSERNAMES"`
+	HostDataInterface string `json:"HOSTDATA"`
+	Vlan              string `json:"VLAN"`
```

Comment:
This struct seems to be mixing global info and host specific info.
Would it be better to split it into two separate structs?